packages/tasks/src/model-libraries.ts

```diff
@@ -263,6 +263,22 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: { term: { path: "model.safetensors" } },
 	},
+	multi_token_prediction: {
```

Comment:
This is the tag that should be added to the model card. Better to use `-` for consistency with other libs 
```suggestion
	"multi-token-prediction": {
```